src/app/zap-templates/templates/app/cluster-typedefs.zapt

```diff
@@ -0,0 +1,53 @@
+{{> header}}
+
+#pragma once
+
+#include <stdint.h>
+
+
+namespace chip {
+namespace app {
+namespace Clusters {
+
+namespace detail {
+// typedefs shared across multiple clusters.
+{{#zcl_typedefs}}
+
+{{#if has_more_than_one_cluster}}
+
+{{> cluster_typedefs_enum ns=""}}
```

Comment:
Where is cluster_typedefs_enum defined?

Reply:
Fixed, should've been cluster_typedefs_typedef